src/Sylius/Bundle/PromotionBundle/Form/Type/Action/PercentageDiscountConfigurationType.php

```diff
@@ -34,6 +34,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'constraints' => [
                     new NotBlank(),
                     new Type(['type' => 'numeric']),
+                    new Range([
+                        'min' => 0.01,
```

Comment:
Also we should add scenario for putting value lower than minimum to see that validation works.